src/explorer.api/Models/ExploreResult.cs

```diff
@@ -26,12 +29,8 @@ public ExploreResult(Guid explorationId, Exploration exploration)
             Status = exploration.Status;
             DataSource = exploration.DataSource;
             Table = exploration.Table;
-            SampleData = exploration.SampleData;
-            Columns = exploration.ColumnExplorations.Select(ce =>
-                new ColumnMetricsCollection(
-                    ce.Column,
-                    Exploration.ConvertToExplorationStatus(ce.Completion.Status),
-                    ce.PublishedMetrics.Select(m => new Metric(m.Name, m.Metric))));
+            SampleData = exploration.SampleData.Select(col => col.ToList()).ToList();
+            Columns = exploration.PublishedMetrics.Select(m => m.Metric).ToList();
```

Comment:
This line is kind of misleading: why the columns are actually metrics?

Reply:
I suppose it should really be called `ColumnMetrics` or simply `Metrics` however the generated json will still need to be `"columns"` for now or we will break the api...

Reply:
Changed it to `ColumnMetrics`
e0fd373